src/allotropy/parsers/beckman_pharmspec/beckman_pharmspec_structure.py

```diff
@@ -0,0 +1,165 @@
+from dataclasses import dataclass
+import re
+
+import pandas as pd
+
+from allotropy.parsers.utils.uuids import random_uuid_str
+
+# This map is used to coerce the column names coming in the raw data
+# into names of the distribution properties.
+COLUMN_MAP = {
+    "Cumulative Counts/mL": "cumulative_particle_density",
+    "Cumulative Count": "cumulative_count",
+    "Particle Size(µm)": "particle_size",
+    "Differential Counts/mL": "differential_particle_density",
+    "Differential Count": "differential_count",
+}
+
+
+VALID_CALCS = ["Average"]
+
+
+@dataclass(frozen=True, kw_only=True)
+class DistributionRow:
+    particle_size: float
+    cumulative_count: float
+    cumulative_particle_density: float
+    differential_particle_density: float | None = None
+    differential_count: float | None = None
+    distribution_row_id: str
+
+
+@dataclass(frozen=True, kw_only=True)
+class Distribution:
+    name: str
+    data: list[DistributionRow]
+    is_calculated: bool
+    distribution_id: str
+
+    @staticmethod
+    def create(df: pd.DataFrame, name: str, *, is_calculated: bool) -> "Distribution":
```

Comment:
Are quotes around "Distribution" required here? Does that cause a `NameError`? I think it shouldnt since the class is defined earlier in the code.

Reason to drop quotes: 
A) type suggesters work better w/o quotes although mypy doesnt seem to care
B) Benchling traditionally we dont use quote for type specification - I think maybe command clicking also fails when quotes are specified?

Reply:
You remove the need for quotes by importing annotations: `from future import __annotaitons__`

Reply:
Fixed.